docs/ide-usage.md

```diff
@@ -64,6 +64,7 @@ The following plugins are needed:
      Configure for all the languages used in TEAMMATES:
      * Java: `Java → Code Style → Formatter → Edit → Tab policy → Spaces only`.
      * JavaScript: `JavaScript → Code Style → Formatter → Edit → Tab policy → Spaces only`.
+     <br>You can find the Web and XML options in `Eclipse → Preferences → ...`.
```

Comment:
I meant to the original version without a `<br>`,  but with a blank line containing trailing spaces 😅